storyblok-vercel-configuration.md

```diff
@@ -0,0 +1,86 @@
+# Production, Staging, Preview configuration
+
+Storyblok = place where content is stored (Headless CMS = content management system)
+
+GitHub (this repo) = place where code is stored (hosted SVC = software version control)
+
+Vercel/Next.js = service that generates and deploys website by combining content + code.
+
+The following document uses Quansight Consulting "LLC" as an example, but it also applies to
+Labs: wherever you come across the string `consulting` below, you just have to
+replace it with `labs`.
+
+## Production = Live Public-Facing Site
+
+Currently deploys to: quansight-consulting.vercel.app
+
+Later, will deploy to: quansight.com
+
+Triggered by:
+
+- adding new commits (e.g. merging a PR) to `main` branch of [Quansight-website
+  repo](https://github.com/Quansight/Quansight-website)
+- Promote to Production button in vercel.com admin
+
+Behind the scenes: Vercel fetches all of the Published content from Storyblok
+and generates a static site using the code on `main`.
+
+## Staging = Semi-Public Preview Site
+
+Semi-public = anybody with the url can see the preview site so be careful not to
+publish anything sensitive or prematurely to staging!
+
+Deploys to: quansight-consulting-git-develop-quansight.vercel.app
```

Comment:
Is this correct?

Reply:
Almost. In fact development branches are deployed to domain with hash: https://quansight-consulting-913gd8ozj-quansight.vercel.app/

We can connect development builds to some static domain. However, you need to remember to clean up cache before testing, while on hashed domain all assets have unique names

Reply:
@trallard based on subsequent conversations with @marcin, I believe your assumption about "cache" meaning local browser cache is correct.